src/main/java/com/sda/entity/Category.java

```diff
@@ -0,0 +1,62 @@
+package com.sda.entity;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "Category")
```

Comment:
If you make table columns with Caps then it would be better to name table also with Caps